packages/utils/src/carequality/cq-monthly-stats/shared.ts

```diff
@@ -30,11 +32,10 @@ export type CQDirectoryEntryData = {
   lat?: number;
   lon?: number;
   state?: string;
-  data?: Organization;
+  data?: FhirOrganization;
```

Comment:
The original type now is FHIR Organization - [reference](https://github.com/metriport/metriport/blob/develop/packages/api/src/external/carequality/cq-directory.ts#L16)